downstreamadapter/worker/producer/kafka_ddl_producer_test.go

```diff
@@ -1,126 +0,0 @@
-// Copyright 2025 PingCAP, Inc.
-//
```

Comment:
This should not be removed completely, this test should be irrelevant to the sarama